lib/ansible/module_utils/acme.py

```diff
@@ -654,7 +654,7 @@ def get_request(self, uri, parse_json_result=True, headers=None, get_only=False,
         else:
             result = content
 
-        if fail_on_error and info['status'] >= 400:
+        if fail_on_error and (info['status'] < 200 or info['status'] >= 400):
```

Comment:
Is it expected that `3xx` HTTP status code are not considered errors? From my understanding of the code there is nothing following the redirect, so I'm wondering if that would be another edge case.